rules/opentitan/defs.bzl

```diff
@@ -180,7 +180,8 @@ def opentitan_test(
         spx_key = None,
         manifest = None,
         exec_env = {},
-        fpga = _fpga_params(),
+        fpga_cw310 = _fpga_params(),
+        fpga_cw340 = _fpga_params(),
```

Comment:
This change should not be required. The `**kwargs` above should allow you to use `fpga_cw310` and `fpga_cw340` in the cryptotest rules.

Can you try reverting the changes in this file and running one of the tests again?

Reply:
Agreed. Turns out changes to `rules/opentitan/defs.bzl` were unnecessary. Testing on CW340 works with this update.

Reply:
Thanks!